hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/messagevalidation/ExceptionHandler.java

```diff
@@ -1,16 +1,15 @@
 package ca.bc.gov.hlth.hnsecure.messagevalidation;
 
-import java.net.UnknownHostException;
-
-import javax.jms.JMSException;
-
```

Comment:
Just curious if these imports were moved around by hand? If you Organize Imports (Ctrl+Shift+O) in Eclipse, they will show up top of the imports as shown here...which is the proper location.